packages/url_launcher/url_launcher/example/lib/main.dart

```diff
@@ -63,6 +63,12 @@ class _MyHomePageState extends State<MyHomePage> {
   }
 
   Future<void> _launchInWebViewOrVC(Uri url) async {
+    if (!await launchUrl(url, mode: LaunchMode.inAppWebView)) {
+      throw Exception('Could not launch $url');
+    }
+  }
+
+  Future<void> _launchInWebViewOrVCWithCustomHeaders(Uri url) async {
```

Comment:
`_launchAsInAppWebViewWithCustomHeaders` (The `WebViewOrVC` is a legacy of the old API surface.)